server/BREAKING.md

```diff
@@ -1,5 +1,12 @@
 > **Note:** These breaking changes are only relevant to the server packages and images released from `./routerlicious`.
 
+## 0.1038 Breaking Changes
+- [aggregate function from `MongoCollection` become async](#aggregate-function-from-MongoCollection-become-async)
```

Comment:
>become [](http://example.com/codeflow?start=45&length=6)

Typo: became

Reply:
Addressed in d4556832bfd0a9c0cb5bde9bbacb337ab63974ee